### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by mbaudin47**
This PR gathers various documentation updates

### CODE REVIEW

**Comment by adutfoy** [comment]
Thanks Michaël for this work that completes the improvement on Taylor expansions.
I just wanted to say that I do not agree to mention da Veiga about the importance factors as da Veiga is not the author of these indices and has just used them in a particular context.
 If you want to cite an article, you can quote the book: 'Structural Reliability Methods', O. Ditlevsen and H.O. Madsen, chapter 8. You can get the pdf on the web: http://od-website.dk/books/OD-HOM-StrucRelMeth-Ed2.3.7.pdf.

**Comment by mbaudin47** [comment]
- That would be a good reference from importance_form.
- A new help page would be dedicated to the importance factors of a linear model. Then the FORM and Taylor importance factors would reference that one.

**Comment by mbaudin47** [comment]
> I just wanted to say that I do not agree to mention da Veiga about the importance factors as da Veiga is not the author of these indices and has just used them in a particular context.
>  If you want to cite an article, you can quote the book: 'Structural Reliability Methods', O. Ditlevsen and H.O. Madsen, chapter 8. You can get the pdf on the web: http://od-website.dk/books/OD-HOM-StrucRelMeth-Ed2.3.7.pdf.

Da Veiga et al. do not use these indices in the book. They introduce these sensitivity indices in the first and only book devoted on this UQ topic. 
What exact equation and page would you want to cite in Ditlevsen&Madsen's  book?

**Review by adutfoy — COMMENTED** [review]
Fine!

**Comment by adutfoy** [comment]
@josephmure  : may you have a look on this PR?

File: python/doc/examples/graphs/plot_graphs_basics.py [hunk 1]
```diff
@@ -226,9 +226,15 @@ def createHSVColor(indexCurve, maximumNumberOfCurves):
...
 # %%
+# Using `~openturns.GridLayout`.
+grid = ot.GridLayout(1, 2)
+grid.setGraph(0, 0, myPDF)
+grid.setGraph(0, 1, myCDF)
+_ = viewer.View(grid)
 
 # %%
-# We create a figure with the `figure` function from Matplotlib, then we add two graphics with the `add_subplot` function.
+# Another method is to create a figure with the `figure` function from Matplotlib,
```
Comment (mbaudin47) [inline python/doc/examples/graphs/plot_graphs_basics.py:236]:
```suggestion
# Another method is to create a figure with the `figure` function from Ma
```

Comment (mbaudin47) [inline python/doc/examples/graphs/plot_graphs_basics.py:236]:
```suggestion
# Another method is to create a figure with the `figure` function from Matplotlib,
```

File: python/doc/theory/data_analysis/aic.rst [hunk 2]
```diff
@@ -46,13 +46,13 @@ likelihood is high enough to justify the number of parameters used.
 In context of small data, there is a substantial risk that AIC select
 models that have too many parameters. In other words, the risk of
 overfitting is important. To tackle such issue, the AICc criterion was
-developed : it consists in evaluating the AIC with a correction term (
-extra penalty) for small data. The formula is as follows :
+developed : it consists in evaluating the AIC with a correction term
```
Comment (mbaudin47) [inline python/doc/theory/data_analysis/aic.rst:49]:
```suggestion
developed : it consists in evaluating the AIC with a correction term
```

File: python/doc/examples/graphs/plot_graphs_basics.py [hunk 3]
```diff
@@ -216,7 +216,7 @@ def createHSVColor(indexCurve, maximumNumberOfCurves):
 # Create matrices of graphics
 # ---------------------------
 #
-# The library does *not* has objects to create a grid of graphics. However, we can use the `add_subplot` function from Matplotlib.
+# The library provides features to create a grid of graphics. However, we can use the `add_subplot` function from Matplotlib.
```
Comment (josephmure) [inline python/doc/examples/graphs/plot_graphs_basics.py:219]: Throughout this examples, I believe "graphic" as a noun should be replaced with "graph". I am not sure "graphic" as a noun exists in English. See https://context.reverso.net/traduction/francais-anglais/graphique

File: python/doc/theory/reliability_sensitivity/taylor_moments.rst [hunk 4]
```diff
@@ -17,96 +17,104 @@ In the remainder, let :math:`\Cov \uX` be the covariance matrix of :math:`\uX`,
...
-
 where :math:`\mat{C} \in \Rset^{n_X \times n_X}` is the input covariance matrix:
 
 .. math::
 
     c_{ij} = \Expect{\left(X_i - \Expect{X_i}\right)\left(X_j - \Expect{X_j} \right)}
 
+for :math:`i, j \in \{1, ..., n_X\}`.
+Notice that the diagonal of the covariance matrix, :math:`c_{ii} = \sigma_i^2`, is
+equal to the variance of the input variable :math:`X_i`.
```
Comment (josephmure) [inline python/doc/theory/reliability_sensitivity/taylor_moments.rst:28]:
```suggestion
Notice that each diagonal element of the covariance matrix :math:`c_{ii} = \sigma_i^2`, is
equal to the variance of an input variable (:math:`X_i`).
```

File: python/doc/theory/reliability_sensitivity/taylor_moments.rst [hunk 5]
```diff
@@ -17,96 +17,104 @@ In the remainder, let :math:`\Cov \uX` be the covariance matrix of :math:`\uX`,
...
 
+.. math::
 
-The expectation and variance of the first-order expansion are:
+    \Var{Y} \approx \sum_{i=1}^{d} \sum_{j=1}^{d}
+        c_{ij} \frac{\partial h}{\partial x_i}(\vect{\mu}) \frac{\partial h}{\partial x_j}(\vect{\mu}).
 
-  .. math::
+In the special case where the input are independent, then
+the covariance expressions simplifies and we get:
```
Comment (josephmure) [inline python/doc/theory/reliability_sensitivity/taylor_moments.rst:55]:
```suggestion
In the special case where the inputs are independent, then
the variance expression is simplified and we get:
```

File: python/src/FunctionalChaosAlgorithm_doc.i.in [hunk 6]
```diff
@@ -98,17 +99,17 @@ as the tensor product of orthonormal univariate families.
...
-      For more details, see :class:`~openturns.StandardDistributionPolynomialFactory`;
+- *polynomials*: the associated distribution :math:`\mu_i` is continuous or discrete. 
+  Note that it is possible to build the polynomial family orthonormal to any univariate 
+  distribution :math:`\mu_i` under some conditions. 
+  For more details, see :class:`~openturns.StandardDistributionPolynomialFactory`;
 
-    - Haar wavelets: they enable to approximate functions with discontinuities.
-      For more details, see :class:`~openturns.HaarWaveletFactory`,;
+- Haar wavelets: they enable to approximate functions with discontinuities.
+  For more details, see :class:`~openturns.HaarWaveletFactory`,;
```
Comment (josephmure) [inline python/src/FunctionalChaosAlgorithm_doc.i.in:108]:
```suggestion
  For more details, see :class:`~openturns.HaarWaveletFactory`;
```

File: python/src/TaylorExpansionMoments_doc.i.in [hunk 7]
```diff
@@ -10,14 +10,14 @@ limitStateVariable : :class:`~openturns.RandomVector`
 
 Notes
 -----
-Assuming that :math:`\uX` has its two first moments finite and that :math:`h` is sufficiently smooth,
-some Taylor expansions of the function :math:`h` are used to approximate the mean and variance
+Assuming that :math:`\uX` has its two finite first moments and that :math:`h` is sufficiently smooth,
```
Comment (josephmure) [inline python/src/TaylorExpansionMoments_doc.i.in:13]:
```suggestion
Assuming that :math:`\uX` has finite first and second order moments and that :math:`h` is sufficiently smooth,
```

**Review by josephmure — CHANGES_REQUESTED** [review]

File: python/doc/examples/graphs/plot_graphs_basics.py (hunk 3 above)
Comment (mbaudin47) [reply python/doc/examples/graphs/plot_graphs_basics.py]: Fixed at a27e17e61

**Comment by mbaudin47** [comment]
@josephmure : I took your suggestions into account. Please review the changes.

File: python/doc/theory/reliability_sensitivity/taylor_moments.rst [hunk 8]
```diff
@@ -17,96 +17,104 @@ In the remainder, let :math:`\Cov \uX` be the covariance matrix of :math:`\uX`,
...
 
-
 where :math:`\mat{C} \in \Rset^{n_X \times n_X}` is the input covariance matrix:
 
 .. math::
 
     c_{ij} = \Expect{\left(X_i - \Expect{X_i}\right)\left(X_j - \Expect{X_j} \right)}
 
+for :math:`i, j \in \{1, ..., n_X\}`.
+Notice that each diagonal element of the covariance matrix :math:`c_{ii} = \sigma_i^2`, is
```
Comment (josephmure) [inline python/doc/theory/reliability_sensitivity/taylor_moments.rst:27]:
```suggestion
Notice that each diagonal element of the covariance matrix :math:`c_{ii} = \sigma_i^2` is
```

**Comment by mbaudin47** [comment]
@josephmure : Please review this PR.

**Review by josephmure — APPROVED** [review]

**Comment by mbaudin47** [comment]
@jschueller : I think that this PR is good to go.

**Pull request merged**